chpl/chpl-resources/src/main/resources/ocd-3302.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<job-scheduling-data
+    xmlns="http://www.quartz-scheduler.org/xml/JobSchedulingData"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.quartz-scheduler.org/xml/JobSchedulingData http://www.quartz-scheduler.org/xml/job_scheduling_data_2_0.xsd"
+    version="2.0">
+    <schedule>
+        <!--  Update 2015 listings as Cures Updated, where appropriate -->
+        <job>
+            <name>addCuresUpdatedToListings</name>
```

Comment:
This is a small thing but the job is "addCuresUpdate**d**" and the logger is "addCuresUpdate" . Not sure which one you want to go with but I would like them to be the same!